smoke-tests/smoke-e2e.bats

```diff
@@ -59,3 +59,10 @@ teardown_file() {
 	result=$(span_attributes_for ${DOCUMENT_LOAD_SCOPE} | jq "select(.key == \"session.id\").value.stringValue")
 	assert_not_empty "$result"
 }
+
+@test "Agent includes SampleRate key on all spans" {
+  result=$(span_attributes_for ${DOCUMENT_LOAD_SCOPE} | jq "select(.key == \"SampleRate\").value.intValue")
+  assert_equal "$result" '"1"
+"1"
+"1"'
+}
```

Comment:
out of scope for this pr, but what do you think about adding a readme to the repo on how to get set up w/ bats? i think it's bread-and-butter for the telemetry team, but i'm not sure what i need to do to get it up & running myself 

i got a kick out of my google attempt:

![Screenshot 2024-02-13 at 9 38 06 AM](https://github.com/honeycombio/honeycomb-opentelemetry-web/assets/14351110/55bf968e-b49d-41fc-a940-887a7eae0e9f)

Reply:
Oh derp you are totally right! I meant to add that when I first added smoke tests but looks like I definitely missed it. Will update.

Reply:
Actually yeah I'll do a quick follow-up PR specific to smoke tests, there's a couple other nits I want to take care of as well.

Reply:
- https://github.com/honeycombio/honeycomb-opentelemetry-web/pull/71